SimDataFormats/Associations/plugins/BuildFile.xml

```diff
@@ -1,5 +1,6 @@
 <use name="FWCore/Framework"/>
 <use name="FWCore/ParameterSet"/>
+<use name="eigen"/>
```

Comment:
hi @lecriste - what triggers the need for this dependency?

Reply:
Hi @davidlange6, that dependency is needed by [this include](https://github.com/cms-sw/cmssw/blob/886d64926c230c57bea1d6c1fdc72e72c19bf497/DataFormats/HGCalReco/interface/Trackster.h#L12).

Reply:
So then its the wrong change. Instead, add a dependency on DataFormats/HGCalReco. Looks like you have also added DataFormats/ForwardDetId and RecoLocalCalo/HGCalRecAlgos that should be added to the BuildFIle.

Reply:
OK, fixed in the latest commit.